src/utils/store/store-actions.ts

```diff
@@ -1,5 +1,18 @@
 import { createAction } from '@reduxjs/toolkit';
 
+//TODO где оставлять JSDoc в данном случае?
+/**
+ * Redux actions related to the store.
```

Comment:
Redux actions are always related to a store, that's why it's ok not to clarify it. Instead I would write, that it's common actions for any app, for example, `Common redux actions, related to an application lifecycle`